apps/explorer/src/app/components/txs/details/order/tx-order-peg.tsx

```diff
@@ -0,0 +1,72 @@
+import { t } from '@vegaprotocol/i18n';
+import { TableCell, TableRow } from '../../../table';
+import type { VegaPeggedReference } from '../liquidity-provision/liquidity-provision-details';
+import { Side, PeggedReferenceMapping } from '@vegaprotocol/types';
+import { useExplorerMarketQuery } from '../../../links/market-link/__generated__/Market';
+import type { ExplorerMarketQuery } from '../../../links/market-link/__generated__/Market';
+import { addDecimalsFormatNumber } from '@vegaprotocol/utils';
+
+export interface TxDetailsOrderProps {
+  offset: string;
+  reference: VegaPeggedReference;
+  marketId: string;
+  side: Side;
+}
+
+export function getSettlementAsset(
```

Comment:
chore (non-blocking): function name does not seem to be doing as it suggests. Suggest getMarketDecimals

Reply:
Good spot. This function totally changed functionality and I lazily didn't rename it